tensilelite/Tensile/KernelWriterBetaOnly.py

```diff
@@ -292,6 +292,26 @@ def kernelBodyBetaOnly(self):
     return kStr
 
 
+  @staticmethod
+  def _getKernelName(solution, btype=None):
+    indexChars = INDEX_CHARS
+    # C dimensions
+    name = "C"
+    for i in range(0, solution._state["ProblemType"]["NumIndicesC"]):
+      name += indexChars[i].lower()
+    name += "_"
+    name += solution._state["ProblemType"]["DestDataType"].toChar()
+    if solution._state["ProblemType"]["GroupedGemm"]:
```

Comment:
Can we have these appends in the same Python file because now all these are separated into different yamls, and the user has to modify a lot when adding a naming parameter.